src/popup/components/Modals/AccountImport.vue

```diff
@@ -99,9 +102,10 @@ export default defineComponent({
       store.commit('setBackedUpSeed');
       props.resolve();
       setTimeout(async () => {
-        await getSdk();
-        store.dispatch('accounts/hdWallet/discover');
-      }, 100);
+        watchUntilTruthy(() => isSdkReady.value).then(
```

Comment:
Why not awaiting the `getSdk` ?

Reply:
https://github.com/aeternity/superhero-wallet/pull/2135#discussion_r1246485890

Reply:
We'll resolve this in the future :)

Reply:
That change doesn't reflect what happened properly - `getSdk` is not working here but previously we were only checking if sdk is ready. Usage of `getSdk` here was a mistake as it wasn't working